server/views/partials/prisonIncentiveLevelEditPartial.njk

```diff
@@ -117,6 +117,7 @@
     id: form.formId + "-" + fieldId,
     name: fieldId,
     value: field.value,
+    suffix: { text: "added every 14 days" },
```

Comment:
did you not want the "per week" suffix for the 2 transfer limits higher up to match the view page?